static/usr/bin/core-desktop-session-wrapper.sh

```diff
@@ -5,18 +5,20 @@
 snap_cmd="$1"
 snap_name="$(echo "$snap_cmd" | cut -d . -f 1)"
 
+session_type=$2
```

Comment:
Being a variable, I think it would be better to put it in uppercase.

Reply:
Even better: in ubuntu session I moved that variable, along with XDG_SESSION_TYPE, into the `snapcraft.yaml` file.
![imagen](https://github.com/user-attachments/assets/f4c26bb4-1c1c-4431-a9d6-933e142eb5df)

Reply:
Are you sure you want it in uppercase? I mean `snap_cmd` and `snap_name` are not, it looked like this script was sticking to: non exported variables lower case, exported variables upper case.

In any case I can't remove it as I need it for the if later on.

Reply:
Ok.